tests/oclc_/test_classify_xml_parser.py

```diff
@@ -0,0 +1,103 @@
+# encoding: utf-8
+
+import json
+from nose.tools import set_trace, eq_
+from .. import (
+    DatabaseTest,
+    sample_data
+)
+from lxml import etree
+from core.model import Contributor, Identifier
+from oclc.classify import OCLCClassifyXMLParser
+
+class TestOCLCClassifyXMLParser(DatabaseTest):
+
+    parser =  OCLCClassifyXMLParser()
+
+    def sample_data(self, filename):
+        return sample_data(filename, 'oclc_classify')
+
+    def tree(self, filename):
+        xml = self.sample_data(filename)
+        return etree.fromstring(xml, parser=etree.XMLParser(recover=True))
+
+    def test_initial_look_up(self):
+        single_tree = self.tree("single_work_response.xml")
+        code, owi_data = self.parser.initial_look_up(self._db, single_tree)
+        eq_(2, code)
+        eq_(1, len(owi_data))
+        eq_(Identifier.OCLC_WORK, owi_data[0].type)
+        eq_(None, owi_data[0].identifier)
+        eq_(1, owi_data[0].weight)
```

Comment:
A lot of code like this can be simplified by doing `[owi] = owi_data`. That will crash unless there's exactly one item in `owi_data`, and then you have the thing you're testing in `owi` and you can work on it from that point.